spec/javascript/support/console.ts

```diff
@@ -19,19 +29,19 @@ export function chaiConsoleSpy(chai, utils) {
     'loggedError',
     (message) => {
       if (message) {
-        const index = console.unverifiedCalls.findIndex((calledMessage) =>
+        const index = (console as ExtendedConsole).unverifiedCalls.findIndex((calledMessage) =>
```

Comment:
looks like we have to cast `console` like 3 times, would it make sense to cast it once?

```ts
const logger = console as ExtendedConsole;
```

Reply:
> looks like we have to cast `console` like 3 times, would it make sense to cast it once?

Yeah, I was trying to avoid confusion by naming `console` anything other than `console`, but you're right that it turned out rather repetitive. I went with your suggestion in 4f15ecb, but then tinkered with it a bit more and landed at 0735056, avoiding assigning to `console` altogether.